### ISSUE TITLE
README.md walkthrough references tech.v3.dataset

### ISSUE THREAD

**Issue by Prometheus77** [techascent/tech.ml.dataset#258]
I'm assuming this is an old name for this project? I tried running the walkthrough and couldn't get anything to work until I switched out tech.v3.dataset for tech.ml.dataset.

**Comment by cnuernber** [comment]
Yes, thanks for the reminder.  What it is is a global search-replace replacing tech.ml.dataset for tech.v3.dataset.  The old namespace was tech.ml.dataset as it matched the project name but over time as the API evolved breaking changes I changed it to tech.v2.dataset and then tech.v3.dataset.

**Comment by Prometheus77** [comment]
Makes sense. I'd submit a PR myself, but not sure which of the `require` statements out to be `ml` vs. `v3`, so I'll leave it to a pro.

**Comment by cnuernber** [comment]
Only the maven deps are tech.ml.dataset.  All the require statements are tech.v3.dataset :-) - but no worries; I will do a once over.
